sql/CMakeLists.txt

```diff
@@ -55,6 +55,14 @@ set(PRE_UPDATE_FILES
   updates/pre-update.sql
 )
 
+# The POST_UPDATE_FILES should be executed as the last part of
+# the update script. 
+# sets state for executing POST_UPDATE_FILES during ALTER EXTENSION
+set(POST_UPDATE_STRING  "set timescaledb.update_script_stage = 'post'\; " )
```

Comment:
Is the backslash at the end intentional?

Will the variable be reset after the update finishes or do we need to manually unset this as the very last step of the update?

Reply:
This is a session variable. So should be unset as well.
Need to escape ; in makefiles, otherwise they won't get printed.

Reply:
session variables are only reset at the end of the session (connection close), so I do worry that this will remain after the update until a user closes the current session.

Reply:
I modified the script to undo the set.  (see CMakeLists.txt)